### DISCUSSION TITLE
Utilização do componente UpLoad

### DISCUSSION THREAD

**Discussion by Smmartinez** [po-ui/po-angular#1001] [Angular]
Opa pessoal, tudo bem? Estou com uma dúvida com relação a utilização do componente UPLOAD.
Eu posso direcionar os arquivos diretamente para um FTP? OU preciso criar um endpoint para receber os arquivos e ai sim colocar no FTP?

Alguém teria alguma exemplo simples?

**Comment by jhonyeduardo** [comment]
Bom dia,

O componente envia Content-Type=Multipart/Form-Data, portanto teria que receber via API para repassar ao FTP.

Uma sugestão seria selecionar os arquivos e fazer o envio de uma outra forma, usando o "ng-model", porém não teria o "progresso" do envio refletido no componente.
